src/Simplex_tree/include/gudhi/Simplex_tree/Simplex_tree_iterators.h

```diff
@@ -455,6 +455,102 @@ class Simplex_tree_skeleton_simplex_iterator : public boost::iterator_facade<
   int curr_dim_;
 };
 
+/** \brief Iterator over the simplices of the simplicial complex that match the dimension specified by the parameter.
```

Comment:
```suggestion
/** \brief Iterator over the simplices of the simplicial complex that matches the dimension specified by the parameter.
```

Reply:
Why? "the simplices [...] match" looks correct.

Reply:
I read this sentence three times to be sure and read a singular every time... You are right, I will delete the comment.